docs/data-export.md

```diff
@@ -6,9 +6,9 @@ To export data out of Scarf, go to the main dashboard and click "Export packages
 
 ![Export packages data](assets/pics/data-export/export-packages-data.png)
 
-The data you can export from Scarf includes all events (defined as package downloads and pixel views) from every user that has interacted with your Scarf-enabled artifacts (packages and pixels). Upon clicking "Export packages data", this data will download as a .csv file. 
+The data you can export from Scarf includes all events (defined as package downloads and pixel views) from every user that has interacted with your Scarf-enabled artifacts (packages and pixels). Upon clicking "Export packages data", this data will download as a .csv file.
 
-You can also export this data using [the Scarf API](https://docs.scarf.sh/api-v2/).
+You can also export this data using [the Scarf API](/api-v2).
```

Comment:
Is this link correct? We are now using api-docs.scarf.sh

Reply:
not anymore... I'll need to update to the new link

Reply:
Ping me once that update is looking good and I'll merge!